admin/bin/validate-gpg-key.sh

```diff
@@ -58,7 +70,16 @@ printf "Validating fingerprint and public key key match...\n"
 printf "\t Public key: %s\n" "${key_location}"
 printf "\t Fingerprint: %s\n" "${fingerprint}"
 
-gpg2 --fingerprint "$fingerprint"
+gpg2 --fingerprint "$fingerprint" || report_error $KEY_MISMATCH
+
+# Validate against Seqouia's key linter, which checks whether OpenPGP certificates
+# use a SHA-1 based binding signature
+# (see https://sequoia-pgp.org/blog/2023/02/01/202302-happy-sha1-day/).
+# Note: it is possible that the key has been updated on another admin workstation,
```

Comment:
This is a problem regardless if admin workstations are out of sync right? Because then running playbooks will try to provision/configure an older key?

Reply:
Yes, but only if admins run `install,` which doesn't happen often. (It's possible an admin stick could only be used for accessing the Admin interface, performing backups etc)